app/helpers/user_helper.rb

```diff
@@ -0,0 +1,10 @@
+module UserHelper 
+
+    # factory methode
+    def subscribe(user)
+        if $SUBSCRIBE_SERVICE.upcase == "SYMPA"
```

Comment:
1. Use `.eql?` instead of  "=="
2. Change the name of `$SUBSCRIBE_SERVICE` to `$ANNOUNCE_LIST_SERVICE`